app/packages/components/src/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import dayjs from 'dayjs';
+import { atom, useRecoilState } from "recoil";
+import TextField from '@mui/material/TextField';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+
+
+interface DatePickerProps {
+  label: string;
+}
+
+const dateValue = atom({
+  key: "dateValue",
+  default: dayjs('2024-01-01'),
+});
+
+const BasicDatePicker: React.FC<DatePickerProps> = ({
+  label,
+}) => {
+  const [value, setValue] = useRecoilState(dateValue);
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
```

Comment:
have we already themed this for the app to match existing components or is the goal of this PR just to add the component and we will tweak colors / formatting once it's added into the sidebar?

Reply:
Let's add the component and we can tweak the colors later